Sources/Public/ImageProvider/AnimationImageProvider.swift

```diff
@@ -17,5 +19,18 @@ import Foundation
 /// Additionally custom Image Providers can be made to load images from a URL,
 /// or to Cache images.
 public protocol AnimationImageProvider {
+
+  /// Whether or not the resulting image of this image provider can be cached by Lottie. Defaults to true.
+  /// If true, Lottie may internally cache the result of `imageForAsset`
+  var cacheEligible: Bool { get set }
```

Comment:
I don't think we want this to be mutable

```suggestion
  var cacheEligible: Bool { get }
```